### ISSUE TITLE
Instruct pytest-cov to ignore ufunc functions

### ISSUE THREAD

**Issue by mhostetter** [mhostetter/galois#11]
Pytest coverage isn't smart enough to know that code is covered when a function is JIT compiled and then called.

```python
# Marks function contents as uncovered
def _add(a, b):
    # Calculate a + b
    return (a + b) % ORDER

_numba_ufunc_add = numba.vectorize(["int64(int64, int64)"], nopython=True)(_add)
# ... other code calls _numba_ufunc_add()
```

**Comment by mhostetter** [comment]
Adding `numba` coverage to `pytest-cov` seems to be a requested feature. https://github.com/numba/numba/issues/4268

**Comment by mhostetter** [comment]
I think `# pragma: no cover` should exclude a function from `pytest-cov`.

https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html

**Comment by mhostetter** [comment]
Implemented in c7a94f537e504aed50f498e955833adb1442d0f1.
